utils/src/utils/AzExtFsExtra.ts

```diff
@@ -73,6 +73,12 @@ export namespace AzExtFsExtra {
         await workspace.fs.writeFile(uri, Buffer.from(contents));
     }
 
+    export async function appendFile(resource: Uri | string, contents: string): Promise<void> {
```

Comment:
maybe a better name is `appendToFile`?

Reply:
Mimicing this https://www.geeksforgeeks.org/node-js-fs-appendfile-function/ so we can have a 1:1 conversion.